docs/hands-on-lab/01-installing-prereqs.md

```diff
@@ -0,0 +1,16 @@
+# Install Puppet Development Kit (PDK) and other tools
+
+To start, install Puppet Development Kit (PDK), which provides all the necessary tools and libraries to build and test modules. We also recommend an emulator for the target device, a code editor with good Ruby and Puppet support, and git — a version control system to keep track of your progress.
+
+1. [Download PDK](https://puppet.com/download-puppet-development-kit) on your platform of choice.
+
+2. If you do not have a Philips HUE hub available, you can download the [Hue-Emulator](https://github.com/SteveyO/Hue-Emulator/raw/master/HueEmulator-v0.8.jar). You need to have Java installed to run this.
```

Comment:
Is hub the right word here? Or is it meant to be bulb? If it's correct please ignore!

Reply:
Assuming it's fine as it's used throughout.

Reply:
In truth, you'd need both the hub and the bulbs.

Reply:
Do we need to say that? Or is it fine in this instance to just say hub?

Reply:
I'll also add "and bulbs" for clarity.